### PR TITLE
Don't delete namespace after kubemark tests, as cluster will be torn …

### PR DESCRIPTION
**Pull request by gmarek**
…down either way

cc @quinton-hoole

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -1436,7 +1436,7 @@ if [[ "${USE_KUBEMARK:-}" == "true" ]]; then
   NUM_MINIONS=${KUBEMARK_NUM_MINIONS:-$NUM_MINIONS}
   MASTER_SIZE=${KUBEMARK_MASTER_SIZE:-$MASTER_SIZE}
   ./test/kubemark/start-kubemark.sh
-  ./test/kubemark/run-e2e-tests.sh && exitcode=0 || exitcode=$?
+  ./test/kubemark/run-e2e-tests.sh --ginkgo.focus="should\sallow\sstarting\s30\spods\sper\snode" --delete-namespace="false"
```
Comment (quinton-hoole) [inline hack/jenkins/e2e.sh]: Preesumably you don't want this "focus" here?  What if some people want to run other e2e tests on Kubemark?

Comment (gmarek) [inline hack/jenkins/e2e.sh]: It's Jenkins configuration, so there's no problem with that.

File: test/kubemark/run-e2e-tests.sh [hunk 2]
```diff
@@ -33,10 +33,10 @@ export KUBE_MASTER_URL="https://${KUBE_MASTER_IP}"
...
 
-if [ -z "$@" ]; then
+if [[ -z "$@" ]]; then
 	ARGS='--ginkgo.focus=should\sallow\sstarting\s30\spods\sper\snode'
 else
 	ARGS=$@
 fi
 
-${KUBE_ROOT}/hack/ginkgo-e2e.sh "--e2e-verify-service-account=false" "$ARGS"
+${KUBE_ROOT}/hack/ginkgo-e2e.sh "--e2e-verify-service-account=false" $ARGS
```
Comment (wojtek-t) [inline test/kubemark/run-e2e-tests.sh]: Why are you removing "" here?

File: hack/jenkins/e2e.sh (hunk 1 above)
Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: I agree that's not a big deal - it's pure replacement of out variables like "***_SKIP_TESTS"

File: test/kubemark/run-e2e-tests.sh (hunk 2 above)
Comment (gmarek) [inline test/kubemark/run-e2e-tests.sh]: Otherwise all ARGS will be passed as a single argument with whitespaces, not a "list" of them

Comment (wojtek-t) [inline test/kubemark/run-e2e-tests.sh]: OK - got it.

**Comment by wojtek-t** [comment]
lgtm

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**